fusion_docs/guide/snapshots.mdx

```diff
@@ -39,6 +39,7 @@ To use Fusion Snapshots in a Seqera AWS Batch compute environment, set the follo
 ```java
 aws.batch.maxSpotAttempts= 5
 fusion.containerConfigUrl = '<CUSTOM_CONTAINER_URL>'
+process.resourceLimits = [cpus: 32, memory: '60.GB']
```

Comment:
In a near future this config section is going to change because it will be possible to enable snapshots only with `fusion.snapshots = true`. I think that it will be better to move this as a tip or example to the section "EC2 instance selection guidelines"

Reply:
This setting is already avail in nextflow 24.10.6 and 25.03.x